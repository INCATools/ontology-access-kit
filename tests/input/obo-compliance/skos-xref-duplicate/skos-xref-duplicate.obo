format-version: 1.4
ontology: skos-xref-duplicate
idspace: skos http://www.w3.org/2004/02/skos/core#

[Term]
id: X:1
relationship: skos:exactMatch Y:1
xref: Y:1

[Typedef]
id: skos:exactMatch
is_metadata_tag: true

