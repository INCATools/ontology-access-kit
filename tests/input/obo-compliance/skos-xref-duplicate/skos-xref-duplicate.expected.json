{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/skos-xref-duplicate.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/X_1",
      "type" : "CLASS",
      "meta" : {
        "xrefs" : [ {
          "val" : "Y:1"
        } ],
        "basicPropertyValues" : [ {
          "pred" : "http://www.w3.org/2004/02/skos/core#exactMatch",
          "val" : "http://purl.obolibrary.org/obo/Y_1"
        } ]
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasDbXref",
      "lbl" : "database_cross_reference",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#id",
      "lbl" : "id",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.w3.org/2004/02/skos/core#exactMatch",
      "type" : "PROPERTY",
      "meta" : {
        "basicPropertyValues" : [ {
          "pred" : "http://www.geneontology.org/formats/oboInOwl#is_metadata_tag",
          "val" : "true"
        } ]
      }
    } ]
  } ]
}